.github/workflows/main.yml

```diff
@@ -0,0 +1,30 @@
+name: Snyk
+
+on:
+  push:
+    branches: [ main, jd-snyk-action ]
+  workflow_dispatch:
```

Comment:
We've added a schedule trigger as well to our workflows.  But I'm not sure if Snyk does scheduled runs anyway so could be redundant

Reply:
it doesn't but is there a benefit to scheduled runs?  I would have thought they could only change if a commit is made?

Reply:
Discovery of a vulnerability in a dependency that's already in use?

Reply:
Actually does that make sense? Presumably Snyk would just update its list so it is redundant.

Reply:
yeah I think (although happy to be corrected) the monitor is just uploading the deps tree and I think the comparing with new vulns will happen regardless

Reply:
It's not 100% clear but the docs suggest that's ok

> Use the snyk monitor CLI command to create a snapshot of a project on the Snyk website that will be continuously monitored for new vulnerabilities.

https://docs.snyk.io/introducing-snyk/snyks-core-concepts/running-tests#:~:text=Use%20the%20snyk%20monitor%20CLI%20command%20to%20create%20a%20snapshot%20of%20a%20project%20on%20the%20Snyk%20website%20that%20will%20be%20continuously%20monitored%20for%20new%20vulnerabilities.